tendrl/node_agent/objects/service/atoms/start.py

```diff
@@ -0,0 +1,25 @@
+from tendrl.common.utils \
+    import service
+
+TENDRL_CLUSTER_ID_PATH_PREFIX = "/var/lib/tendrl/"
+
+
+class Start(object):
+    def run(self, parameters):
+        # write the cluster_id to cluster_id file
+        cluster_id_path = TENDRL_CLUSTER_ID_PATH_PREFIX + parameters.get(
+            "Tendrl_context.sds_name") + "_cluster_id"
```

Comment:
My feeling is that, this atom would be more useful if it just performed the task of starting a service(any service). 

My understanding is that, atoms are not directly exposed to outside world, its only that they are exposed via a flow and that IDEALLY is exposed ONLY via an api.

So, given that, can this atom be made generic to start any service and only start any service(can the cluster id part move else where), as I see a good use of such a generic atom. Else, if such a generic module already exists then shouldn't this be the like the flow does in its 
* pre section, the cluster id storage into a file part
and probably start the service using the generic already existing module as part of pre or post sections as applicable? Just that I am not sure if this atom is actually atomic :)

Note: Correct me if my understanding of the exposure of atom is incorrect.